news_scraper/spiders/businessstandard.py

```diff
@@ -34,6 +34,7 @@ def parse(self, response):
         article.add_xpath(
             "article_text",
             '//div[@id="parent_top_div"]/div/text() | //div[contains(@class, "MainStory_storycontent__Pe3ys") and contains(@class, "storycontent")]/p/text() | //div[contains(@class, "MainStory_storycontent__Pe3ys") and contains(@class, "storycontent")]//div/text()',
+            # multiple selectors included to robustly handle data extraction across urls
```

Comment:
@ARAVINT-S-A The comment should be above the selectors. I clearly mentioned in the slack post that you have to explain why we are using multiple selectors. You'll have to say something like.. "when the URL is loaded from a EU proxy (for eg. with https://www.proxysite.com/) , the server is using CSS selectors for the article content rather than the Indian version"